### DISCUSSION TITLE
For live tiles are not updating under 24H2

### DISCUSSION THREAD

**Discussion by XiaoTDM** [valinet/ExplorerPatcher#3963] [Q&A]
Live tiles are known to be unstable at **24H2**, which in turn causes tile updates to be completely random.
It seems that everyone is reluctant to fix it.
But I'm a heavy live tiles reliant + OCD, and the tiles that used to be available are now static, which makes me very uncomfortable and sad. This also indirectly caused **Live Tiles Anywhere** to not work, which was a headache for me.
I'm sure **I'm not the only one who used EP just because It's able to restore live tiles**.
Could request a repair attempt? Thanks very much!

**Comment by canapaio** [comment]
why did microsoft come down so hard on your excellent programme?

**Reply by XiaoTDM** [reply]
Quite simply, these modifications often touch the underlying code of the system, resulting in more or less instability. Microsoft disables these mods in order to avoid user complaints and security ~~(and design concerns)~~.

**Reply by Amrsatrio** [reply]
This program used to be merely something that unearths remnant code that are totally expected to be nuked in future versions. Both the 10 taskbar and 10 start menu are now no longer part of the latest versions of Windows 11.

**Reply by XiaoTDM** [reply]
How long has a phone dialer been retained, but, the start menu have all been overturned rapidly.💦

**Comment by Amrsatrio** [comment]
I would, in 2025 or 2026. I plan to study the backend code for this just in case they are preparing to nuke this further. I request you two things: please be patient and lower your expectations.

**Reply by XiaoTDM** [reply]
Good luck with you! (Actually, I'm now passively used to the start menu of the new W11.😅)
